Pod/Classes/LSClockState.h

```diff
@@ -1,15 +1,15 @@
 #import <Foundation/Foundation.h>
 NS_ASSUME_NONNULL_BEGIN
 
-/// A straight port/copy of the `rl-cruntime-common` ClockState javascript prototype.
-@interface LSClockState : NSObject
+@protocol LSTimeProvider
+- (NSDate *)currentTime;
+- (SInt64)offsetInMicroseconds;
```

Comment:
One problem is that API requires it here:
https://github.com/supermarin/lightstep-tracer-objc/blob/1772146ddbb6e990780504deeb02f0b8a2d5bb8a/Pod/Classes/LSTracer.m#L317

Is the parameter required or optional?